src/ai/backend/client/func/keypair_resource_policy.py

```diff
@@ -102,7 +102,7 @@ async def update(
         max_vfolder_count: int,
         max_vfolder_size: int,
         idle_timeout: int,
-        allowed_vfolder_hosts: Sequence[str],
+        allowed_vfolder_hosts: str = None,
     ) -> dict:
```

Comment:
Here as well. The type definition of `total_resource_slots` is `str`, not `int`.